src/vcpkg/tools.cpp

```diff
@@ -549,6 +560,55 @@ namespace vcpkg
         }
     };
 
+    struct Python3Provider : ToolProvider
+    {
+        virtual bool is_abi_sensitive() const override { return false; }
+        virtual StringView tool_data_name() const override { return Tools::PYTHON3; }
+        virtual std::vector<StringView> system_exe_stems() const override { return {"python3", "py3", "python", "py"}; }
+        virtual std::array<int, 3> default_min_version() const override { return {3, 10, 2}; }
```

Comment:
```suggestion
        virtual std::array<int, 3> default_min_version() const override { return {3, 5, 0}; } // 3.5 added -m venv
```

Originally I used 3.10.2 because that's what `vcpkgTools.xml` says, but this breaks MacOS since it's bundled with 3.9.